### DISCUSSION TITLE
Can't pair Blueretro N64 with phone

### DISCUSSION THREAD

**Discussion by Megademon749** [darthcloud/BlueRetro#619] [Q&A]
First Of all thanks for this amazing mod.
When i try to pair any phone or any PC with The ESP32 It doesnt pair.
I can pair controllers to It, but i can't pair any PC or phone.

**Comment by Gkjasl** [comment]
Do you have any Bluetooth devices connected to your phone? My pixel watch was blocking my blueretro internal gamecube adapter from pairing until I turned Bluetooth off on the watch.
